go.mod

```
@@ -1,6 +1,7 @@
 module github.com/IBM/cloudant-go-sdk
 
-go 1.22
+go 1.22.0
+toolchain go1.23.0
```

is there a patch version of the toolchain for the latest security fix?
```suggestion
toolchain go1.23.1
```

---

done in [d4e1283](https://github.com/IBM/cloudant-go-sdk/pull/535/commits/d4e128382cdf45bc6430b24f309c426ec0b0fd88)